### DISCUSSION TITLE
[데모데이] 점심 메뉴

### DISCUSSION THREAD

**Discussion by stockton66785** [team-peekabook/Peekabook-Gidi#28] [Announcements]
안녕하세요

맨날 가라 이슈만 파다가 드디어 issue다운 issue를 파게 되었습니다

데모데이날 점심을 먹어야 하는데요
미리 예약 또는 준비를 해야할 것 같아서요

아래 있는 선택지를 잘 읽어보시고 의견 부탁드립니다

1. 김치찌개
[](https://naver.me/GLKSd4dS)

2. 쌀국수
[](https://naver.me/xevRWlZy)

3. 농민백암순대국
[](https://naver.me/xQOiMUpb)

4. 돈가스
[](https://naver.me/xjeEDjAW)

5. 덕복이
[](https://naver.me/xjejaOYR)

6. 집에서 도시락 싸가기
[](https://peekabook.kr)

**이상**입니다

자세한 건 저녁 스크럼 때 다시 얘기해봅시다
그럼 오늘도 화이팅.

**Comment by devxsby** [comment]
스윙스 하나요

**Reply by HyunwooisMikeSteven** [reply]
돈가스윙스리랑카센타이어차피우승은스윙스
